L1Trigger/TrackTrigger/src/L1TrackQuality.cc

```diff
@@ -159,7 +159,9 @@ void L1TrackQuality::setL1TrackQuality(TTTrack<Ref_Phase2TrackerDigi_>& aTrack)
     }
 
     else if (this->qualityAlgorithm_ == QualityAlgorithm::GBDT) {
-      aTrack.settrkMVA1(ortoutputs[1][1]);
+      if (this->featureNames_.back() == "d0"){ // temp fix, need better way to fill disp MVA
```

Comment:
This really should have a better solution. I would suggest passing the output value through the function and setting the MVA variable in L1FPGATrackProducer.cc with an extended tracking flag since we shouldn't run this BDT unless extended tracking is running because d0 isn't filled.

Reply:
I've implemented your suggestion in commit [dbf8525](https://github.com/cms-l1t-offline/cmssw/pull/1257/commits/dbf852576508f471657cead5177e4066460d8381)